src/types.ts

```diff
@@ -35,6 +35,12 @@ export interface IFtpDeployArguments {
      */
     "dry-run"?: boolean;
 
+    /**
+     * Tries to sync posix file modes to server. Only works for new and updated files.
```

Comment:
Mention this only works for unix based operating systems as well

Reply:
Are you sure stat doesn't work on windows? Nothing in the node docs about it not working https://nodejs.org/api/fs.html#fspromisesstatpath-options and it's defined in libc https://codeforwin.org/2018/03/c-program-find-file-properties-using-stat-function.html so it should work on windows.

Reply:
The client would load the permissions from the files. But the destination FTP server will have to be unix based in order for the `CHMOD` command to work.

Reply:
According to the docs I linked to earlier, it only works on FTP servers that support `CHMOD` which is not the same as unix based FTP servers. There are both non unix based servers that support that command and unix based servers that don't support that command. That's why I write "tries to" rather than "sets" ;)